docs/source/tutorial/visualise_pipeline.md

```diff
@@ -233,3 +235,80 @@ You can view the larger visualisation of the chart by clicking the 'Expand Plotl
 
 
 ![](../meta/images/pipeline_visualisation_plotly_expand.png)
+
+
+### Visualise Matplotlib charts in Kedro-Viz
+
+[Matplotlib](https://matplotlib.org/) is a Python library for creating static, animated, and interactive visualizations. Integrating Matplotlib into Kedro-Viz allows you to output your charts as part of your pipeline visualisation.
+
+You can view Matplotlib charts in Kedro-Viz when you use the [Kedro MatplotLibWriter dataset](https://kedro.readthedocs.io/en/latest/kedro.extras.datasets.matplotlib.MatplotlibWriter.html). You must update the `requirements.txt` file in your Kedro project by adding the following dataset to enable Matplotlib for your project:
+
+`kedro[matplotlib.MatplotlibWriter]==0.18.2`
+
+>**Note:** The MatplotlibWriter dataset converts Matplotlib objects to image files. This means that Matplotlib charts within Kedro-Viz are static and not interactive, unlike the Plotly charts seen above.
+
+<!-- Add example on how to add Matplotlib charts-->
+
+To use this dataset, configure your plot in your Kedro node.
+
+Below is an example of how to visualise plots on Kedro-Viz using `matplotlib.MatplotlibWriter`.
+
+The below functions can be added to the `nodes.py` and `pipeline.py` files respectively.
+
+```python
+# nodes.py
+import matplotlib.pyplot as plt
+
+
+def create_confusion_matrix(companies: pd.DataFrame):
```

Comment:
```suggestion
def create_confusion_matrix(companies: pd.DataFrame):
    actuals = [0,1,0,0,1,1,1,0,1,0,1]
    predicted = [1,1,0,1,0,1,0,0,0,1,1]
    data = {"y_Actual": actuals, "y_Predicted": predicted}
    df = pd.DataFrame(data, columns=["y_Actual", "y_Predicted"])
    confusion_matrix = pd.crosstab(
        df["y_Actual"], df["y_Predicted"], rownames=["Actual"], colnames=["Predicted"]
    )
    sn.heatmap(confusion_matrix, annot=True)
    return plt
```

Reply:
we could remove the random calc from the docs -- that was just for demo website to generate random plots for every run so we can distinguish one run from another.